applications/TrilinosApplication/custom_python/add_trilinos_convergence_accelerators_to_python.cpp

```diff
@@ -53,6 +53,7 @@ void AddTrilinosConvergenceAcceleratorsToPython(pybind11::module &m)
     using TrilinosBaseConvergenceAcceleratorType = ConvergenceAccelerator<TrilinosSparseSpaceType, TrilinosLocalSpaceType>;
 
     py::class_<TrilinosBaseConvergenceAcceleratorType, typename TrilinosBaseConvergenceAcceleratorType::Pointer>(m, "TrilinosConvergenceAccelerator")
+        .def(py::init<>())
```

Comment:
I would not allow this
why do you want to add it?

IMO the only CTor we have should be the one that accepts `Parameter`s

Reply:
The usage will manifest in the next PR, which in order to make possible the usage of the Interface Block Quasi-Newton from Python refactors the Python solvers completely. I will also take the chance to simplify a lot the parallel solvers. These, in order to be in line with the serial ones will use the `FSICouplingInterface`.

The `FSICouplingInterface` owns the convergence accelerator as well as the interface model part used for the residual minimization. This is fine in serial, however the MPI convergence accelerators require the interface model part to be passed by constructor, meaning that we have a sort of interdependency.

In order to overcome this, I decided to create a default MPI communicator in the constructor of the structure `FSICouplingInterface`. Then, once the final convergence accelerator is created, it is assigned to the `FSICouplingInterface `.

(I think it would have been better to add this change in the next PR).

Reply:
could you take out this change from this PR so that we can discuss it in the upcoming one?

Reply:
Sure.